src/nfa/nfa.cc

```diff
@@ -322,6 +322,37 @@ void Delta::add(State state_from, Symbol symbol, State state_to) {
     }
 }
 
+void Delta::add_set(State state_from, Symbol symbol, const StateSet& states) {
```

Comment:
Use `const` for possible optimizations from the compiler.
```suggestion
void Delta::add_set(const State state_from, const Symbol symbol, const StateSet& states) {
```